policyengine_us/parameters/gov/irs/credits/retirement_saving/qualified_retirement_savings_contributions.yaml

```diff
@@ -0,0 +1,22 @@
+description: The IRA sums the following elements as qualified retirement savings contributions when computing the retirement saving credit.
+values:
+  2018-01-01:
+    - traditional_ira_contributions
+    - traditional_401k_contributions
+    - traditional_403b_contributions
+    - roth_ira_contributions
+    - roth_401k_contributions
+    - roth_403b_contributions
+    - self_employed_pension_contributions
```

Comment:
```suggestion
    - self_employed_pension_contributions
    - able_contributions
```

Reply:
we need to create a new variable called `able_contributions_person` and use an adds in the existing `able_contributions`